projects/addin-client/src/src/addin-client.service.ts

```diff
@@ -94,6 +117,12 @@ export class AddinClientService {
         },
         updateContext: (context: any) => {
           this.updateContext.emit(context);
+        },
+        themeChange: (settings: AddinClientThemeSettings) => {
+          themeService.setTheme(new SkyThemeSettings(
```

Comment:
Do we need to sanity check that `settings` is defined?

Reply:
Just to be safe, I think it'll be a good idea to sanity check the settings here.